examples/cifar10/start_fl_poc.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# add current folder to PYTHONPATH
+export PYTHONPATH="${PYTHONPATH}:${PWD}"
+echo "PYTHONPATH is ${PYTHONPATH}"
+
+algorithms_dir="${PWD}/job_configs"
+servername="localhost"
+workspace="${PWD}/workspaces/poc_workspace"
+admin_username="admin"  # default admin
```

Comment:
line 6 and 9 seems not used in this script

Reply:
Good catch. Removed.